start.sh

```diff
@@ -1,3 +1,36 @@
 #!/bin/bash
 
-mitmproxy -s filter.py
+echo -e " 
+Afin de garder le proxy WEB permanent, il serait bien de le faire tourner dans un 'screen' avec:
```

Comment:
or as a proper system unit file ? 

systemd unit are really easy to write, see https://unix.stackexchange.com/questions/15348/writing-basic-systemd-service-files